### DISCUSSION TITLE
Monetisation from GitHub Pages

### DISCUSSION THREAD

**Discussion by ayushch80** [community/community#23456] [Other Feature Feedback, Questions, & Ideas]
<p>Hey Friends, I wanna ask if that can i monetize a website hosted on github pages. :slight_smile:</p>

**Comment by yoannchaudet** [comment]
<p>Hello <a class="mention" href="/ayushch80">@ayushch80</a> and welcome here!</p>
<p>No, GitHub Pages is for <a href="https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#prohibited-uses">non-commercial use only</a> so monetization is typically something that is not allowed.</p>
